app/src/main/AndroidManifest.xml

```diff
@@ -1,131 +1,193 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.gdg.frisbee.android" android:versionCode="27" android:versionName="1.0.10" android:installLocation="auto">
+  package="org.gdg.frisbee.android"
+  android:installLocation="auto">
 
-    <uses-feature android:name="android.hardware.nfc" android:required="false" />
+  <uses-feature
+    android:name="android.hardware.nfc"
+    android:required="false" />
 
-    <supports-screens android:normalScreens="true" android:smallScreens="true" android:largeScreens="true" android:xlargeScreens="true"/>
-    <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.USE_CREDENTIALS"/>
-    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-permission android:name="android.permission.WAKE_LOCK" />
-    <uses-permission android:name="android.permission.NFC" />
-    <uses-permission android:name="android.permission.GET_TASKS" />
+  <supports-screens
+    android:normalScreens="true"
+    android:smallScreens="true"
+    android:largeScreens="true"
+    android:xlargeScreens="true" />
 
-    <permission android:name="org.gdg.frisbee.android.permission.MAPS_RECEIVE" android:protectionLevel="signature"/>
-    <permission android:name="org.gdg.frisbee.android.permission.C2D_MESSAGE" android:protectionLevel="signature" />
-    <uses-permission android:name="org.gdg.frisbee.android.permission.MAPS_RECEIVE"/>
-    <uses-permission android:name="org.gdg.frisbee.android.permission.C2D_MESSAGE" />
-    <uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES"/>
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+  <uses-permission android:name="android.permission.USE_CREDENTIALS" />
+  <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+  <uses-permission android:name="android.permission.WAKE_LOCK" />
+  <uses-permission android:name="android.permission.NFC" />
+  <uses-permission android:name="android.permission.GET_TASKS" />
 
-    <uses-feature
-        android:glEsVersion="0x00020000"
-        android:required="true"/>
```

Comment:
I think google maps requires it. I am not sure the app uses Google Maps V2 SDK though.